tasks/test.py

```diff
@@ -302,6 +302,32 @@ def command(module_results, module, module_result):
 
     return test_core(modules, flavor, ModuleLintResult, "golangci_lint", command)
 
+def build_stdlib(
+    ctx,
+    build_tags: List[str],
+    cmd: str,
+    env: Dict[str, str],
+    args: Dict[str, str],
+    test_profiler: TestProfiler,
+):
+    """
+    Builds the stdlib with the same build flags as the tests.
```

Comment:
Could you expand the description a bit to explain why we need to do this?

Reply:
Added some context in https://github.com/DataDog/datadog-agent/pull/17744/commits/dd78008e32da94f86f633a69775cb801b483d200. Please let me know if this is good 🙏